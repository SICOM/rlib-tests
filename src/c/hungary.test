#
# This test crashes RLIB 2.0.x because it uses:
#
# rlib_set_output_parameter(r, "pdf_fontname", "Liberation Mono");
#
# RLIB 2.0.x doesn't handle fonts that are not in the 14 base fonts.
#

COMMAND=./hungary
FORMATS="pdf"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
