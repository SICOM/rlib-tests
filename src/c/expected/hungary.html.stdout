<?xml version="1.0" encoding="ISO8859-2"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<style type="text/css">
pre { margin:0; padding:0; margin-top:0; margin-bottom:0; font-size:12pt;}
body { background-color: #ffffff;}
TABLE { border: 0; border-spacing: 0; padding: 0; width:100%; }
</style>
<title>RLIB Report</title></head>
<body>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="22" style="font-size: 13px; ">ÁRVÍZTÛRÕ TÜKÖRFÚRÓGÉP</span></pre></div>
<div><pre style="font-size: 1pt"><span data-col="0" data-width="22" style="font-size: 13px; ">árvíztûrõ tükörfúrógép</span></pre></div>
<table cellspacing="0" cellpadding="0" style="width:100%;"><tr><td style="height:4px; background-color:#ffffff; width:100%"></td></tr></table>
<table><!--start from part table--><tr><!--start from part tr--><td><!--started from part td--><!--start pages across--></td><!--ended from part td--><!--end pages across--></tr><!--ended from part tr--></table><!--ended from part table--><div><pre style="font-size: 1pt"><span data-col="0" data-width="6" style="font-size: 11px; ">Page: </span><span data-col="0" data-width="3" style="font-size: 11px; ">  1</span></pre></div>
</body></html>
